dffml/df/base.py

```diff
@@ -86,6 +88,18 @@ def config(self):
         object associated with this operation implementation context.
         """
 
+    @asynccontextmanager
+    async def subflow(self, dataflow):
+        """
+        Registers subflow `dataflow` with parent flow and returns an instance of `BaseOrchestratorContext`
+
+        >>> async with self.subflow(self.config.dataflow) as octx:
+            octx.run(test_inputs)
```

Comment:
This is going to cause a doctest error (and will fail the CI once #431 is fixed) 

How about:

```python
        >>> async def my_operation(arg):
        ...     async with self.subflow(self.config.dataflow) as octx:
        ...         return octx.run({"ctx_str": []})
```